src/IFrameNavigator.ts

```diff
@@ -107,6 +103,7 @@ export default class IFrameNavigator implements Navigator {
     private scroller: ScrollingBookView | null;
     private annotator: Annotator | null;
     private settings: BookSettings;
+    private eventHandler: EventHandler | null;
```

Comment:
That's true. I think I planned to make it nullable and didn't. That wouldn't make sense with a paginator, but if you used it with only a scrolling view it would be ok. I can come back to that later if it's ever useful.